### PR TITLE
feat(REACH-574): Allow passing any attributes to the button for popover and sidetab

### LINKED ISSUE #578: No accessibility implementation
It is important to implement descriptions for users with blindness using title or aria-label description

### PR DESCRIPTION
**Pull request by mathio**
Since the button is generated by the SDK we need to allow passing any attributes to the button for further customization, eg. to add aria-label attribute.

Resolves #578

### CODE REVIEW

**Comment by typeform-ops-gha** [comment]
*[BOT]* Preview available with hash `d6b46f0d7a39c2463909f9dc0df4ac6862958bd8` [here](https://embed.typeform.com/d6b46f0d7a39c2463909f9dc0df4ac6862958bd8/embed.js).

**Comment by typeform-ops-gha** [comment]
*[BOT]* Preview available with hash `e1d8b568aa37f469d2941cbeff81483995125dcb` [here](https://embed.typeform.com/e1d8b568aa37f469d2941cbeff81483995125dcb/embed.js).

File: packages/embed/src/base/partial-element-with-additional-attributes.ts [hunk 1]
```diff
@@ -0,0 +1,10 @@
+interface AdditionalAttributes {
+  // per docs: https://developer.mozilla.org/en-US/docs/Web/API/HTMLElement/style
+  // setting style as string is not advised, but perfectly valid approach
+  style: string
+
+  // data attributes
+  [key: `data${string}`]: string
```
Comment (mumpo) [inline packages/embed/src/base/partial-element-with-additional-attributes.ts:7]: 😮

**Review by mumpo — APPROVED** [review]

**Pull request merged**